Nos/Service/CurrentUser.swift

```diff
@@ -245,6 +245,8 @@ enum CurrentUserError: Error {
         }
     }
     
+    // You probably shouldn't use this. It's slow and blocks the main thread. Try to use `SocialGraphCache.follows(:)`
+    // instead.
```

Comment:
Could we turn this into a doc comment? I'd suggest marking it as deprecated, too, but I'd rather not have a warning in our code indefinitely, so maybe not. Or could we replace all uses of this with `SocialGraphCache.follows(:)`? I think there are around 4.

```suggestion
    /// You probably shouldn't use this. It's slow and blocks the main thread. Try to use `SocialGraphCache.follows(:)`
    /// instead.
```

Reply:
Whoops, just read the rest of your description and realize you were intending to avoid this kind of refactoring here. I'd be willing to give this one a shot if you're ready to be done here. I'm not sure about the other; haven't looked at how much it's used yet.